drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -0,0 +1,154 @@
+{% load i18n %}
+
+<div class="xblock--drag-and-drop--editor editor-with-buttons">
+    {{ js_templates|safe }}
+
+    <section class="drag-builder">
+        <div class="tab feedback-tab">
+            <p class="tab-content">
+                {% trans "Note: don't edit the question if students already answered it! Delete it and create a new one." %}
```

Comment:
"Note: do not edit the problem if students have already completed it. Delete the problem and create a new one."